.circleci/config.yml

```diff
@@ -396,18 +396,7 @@ jobs:
                   killall touristd
                   killall UserNotificationCenter
 
-                  LOGDIR=$(echo "$(pwd)/test/logs")
-                  mkdir -p "$LOGDIR"
-                  APPLOG=$(echo "$LOGDIR/app.log")
-                  DRIVERLOG=$(echo "$LOGDIR/chromedriver.log")
-
-                  pushd "release/mac/WordPress.com.app/Contents/MacOS" 
-                  eval './WordPress.com --disable-renderer-backgrounding --disable-http-cache --start-maximized --remote-debugging-port=9222 > "$APPLOG" 2>&1 &' && sleep 5
-                  popd
-                  
-                  ./node_modules/.bin/chromedriver --port=9515 --verbose > "$DRIVERLOG" 2>&1 &
-                  ./node_modules/.bin/mocha test/tests/e2e.js
-                  osascript -e 'quit app "WordPress.com"'
+                  make e2e
```

Comment:
should we just call `npm run e2e` here instead of calling through `make`?

Reply:
I think it might be nice to have the option of calling the tests either way. There's also some benefit to exposing/documenting critical workflow tooling in the makefile. In addition, all paths in the test script are absolute, which, combined with `make`, gives the developer flexibility in how they are invoked. (e.g. tweaking something from within the Calypso submodule and invoking with `make -f ./Makefile e2e`).